playbooks/molecule/resources/xnat/verify.yml

```diff
@@ -16,3 +16,63 @@
           - response.status == 200
           - response.server == "nginx"
           - response.content is search('MIRSG XNAT')
+
+- name: Verify correct Docker version is installed
+  hosts: xnat_cserv
+  tasks:
+    - name: Get Docker server version
+      ansible.builtin.command:
+        cmd: "{% raw %}docker version -f '{{.Server.Version}}'{% endraw %}"
+      changed_when: false
+      register: server_version
+
+    - name: Get Docker client version
+      ansible.builtin.command:
+        cmd: "{% raw %}docker version -f '{{.Client.Version}}'{% endraw %}"
+      changed_when: false
+      register: client_version
+
+    - name: Check Docker server version
+      ansible.builtin.assert:
+        that:
+          - server_version.stdout is version("24.0.9")
+          - client_version.stdout is version("24.0.9")
+
+- name: Verify container service is running
+  hosts: localhost
+  tasks:
+    - name: Check container service is running
+      ansible.builtin.uri:
+        url: http://localhost:8080/xapi/docker/server/ping
+        user: "{{ xnat_service_admin.username }}"
+        password: "{{ xnat_service_admin.password }}"
+        method: GET
+        validate_certs: false
+        status_code:
+          - 200
+
+    - name: Upload sample image to the Container Service
+      ansible.builtin.uri:
+        user: "{{ xnat_service_admin.username }}"
+        password: "{{ xnat_service_admin.password }}"
+        url: http://localhost:8080/xapi/docker/pull?image=xnat/dcm2niix:latest&save-commands=true
+        method: POST
+        validate_certs: false
+        status_code:
+          - 200
+
+    - name: Get all Docker images
+      ansible.builtin.uri:
+        url: http://localhost:8080/xapi/docker/images
+        user: "{{ xnat_service_admin.username }}"
+        password: "{{ xnat_service_admin.password }}"
+        method: GET
+        validate_certs: false
+        status_code:
+          - 200
+      register: response
+
+    - name: Check for sample image
+      ansible.builtin.assert:
+        that:
+          - response.json[0].tags[0] == "xnat/dcm2niix:latest"
```

Comment:
Are the quotes needed here?

Reply:
yeah otherwise it fails with:
```yaml
fatal: [localhost]: FAILED! =>
  msg: 'The conditional check ''response.json[0].tags[0] == xnat/dcm2niix:latest'' failed. The error was: template error while templating string: expected token ''end of statement block'', got ''latest''. String: {% if response.json[0].tags[0] == xnat/dcm2niix:latest %} True {% else %} False {% endif %}. expected token ''end of statement block'', got ''latest'''
```

Reply:
Cheers!